AGPs/AGP-Comprehensive-Aragon-Security-Review-Program.md

```diff
@@ -0,0 +1,30 @@
+---
+AGP: N/A
+Title: Comprehensive Aragon Security Review Program
+Author: burrrata (@burrrata)
+Status: Stage III
+Track: Association
+Created: 2019-10-02
+---
+
+# AGP-X: Comprehensive Aragon Security Review Program
+
+## Description of desired Association policy change
+
+This proposal is to expand the [Aragon Security Review](https://github.com/aragon/security-review/) program to include all Nest and Flock projects. This will allow developers to focus on building thier applications. It will also allow the Aragon Association to build relationships with auditors, reducing time and money spent on security audits overall. This is essential for Aragon to be able to provide professional and trusted solutions for organizations on a global scale.
+
+Aragon app developers (Nest or Flock) would be required to submit documentation along with a frozen codebase to the AA. At that point the AA would reveiw and, if satisfactory, hand off the project to a security partnet. The security partner would then review, present findings, and provie suggestions for improvement. The Aragon app developers would then need to incorporate all major and critical suggestions provided by the security partner before receiving their ANT bonus. 
```

Comment:
I think the security DAO that @LouisGrx is setting up should be mentioned as i'd say that other people from the ecosystem should evaluate whether a codebase is ready for a productive audit

Reply:
I've literally never heard of the "security DAO" and I check the Aragon forum on a daily basis and I'm one of the most engaged members in the NestDAO WG. Can you please provide a link to more details on this "security DAO?"

Reply:
FTR the newest version of this AGP can be found here:
- https://github.com/aragon/AGPs/pull/117

Reply:
> security DAO

This refers to https://github.com/aragon/security-review/#step-3-flock-securitas-wip